webpack/components/Table/MainTable.js

```diff
@@ -41,7 +33,7 @@ const MainTable = ({
   }
 
   const tableProps = { cells, rows, ...extraTableProps };
-  if (composable) {
+  if (React.Children.count(children) > 0) {
```

Comment:
React.Children?.count

Reply:
Switched to `if(children)` like we talked about